config/application.rb

```diff
@@ -41,7 +42,8 @@ class Application < Rails::Application
     # Do not generate system test files.
     config.generators.system_tests = nil
 
-    config.exceptions_app = self.routes
+    # Handle exceptions not caught by controllers, e.g. db errors
+    config.exceptions_app = ->(env) { RackExceptionsApp.call(env) }
```

Comment:
would this change the behaviour of routing to our normal error pages like 500 etc?

Reply:
404s etc. should behave like before. The new page, however, will be shown for all uncaught exceptions, which means it replaces the internal server error page. Maybe we should return a `500` in light of this?

Reply:
Our `422` page seems to be broken anyway — even without this change?